components/KebijakanPrivasiKetentuanPengguna/ContentDetail.vue

```diff
@@ -0,0 +1,77 @@
+<template>
+  <div>
+    <div class="container mx-auto mt-[16px] md:mt-[38px]">
+      <div class="mx-[16px] md:mx-10">
+        <div class="mb-[22px]">
+          <Breadcumbs :title="dataJson.name" :link="dataJson.link" />
+        </div>
+        <component :is="dataJson.icon" class="icon" />
+        <h1 class="uppercase font-roboto font-bold mt-[16px] text-[20px] dark:text-[#E9E9E9]">
+          {{ dataJson.name }}
+        </h1>
+        <div class="flex items-center rounded-lg bg-gray-50 dark:bg-[#292C2A] mt-[16px] w-fit h-fit py-[8px] pl-[10px] pr-[8px]">
+          <div class="flex-shrink-0">
+            <IconCalendar />
+          </div>
+          <p class="ml-2 text-[12px] min-w-0 flex-1 dark:text[#AAB0B7]">
+            Diperbaharui per tanggal <b class="text-[12px] dark:text-[#E9E9E9]">{{ dataJson.updated }}</b>
```

Comment:
minor:

Hello @marsellavaleria19 , I recommend using `<strong>` tag instead of `<b>` tag if you want to highlight and emphasize words. It has better accessibility for screen reader and help people with disability understand the context better.

ref: https://stackoverflow.com/questions/271743/whats-the-difference-between-b-and-strong-i-and-em

Reply:
Hello @Ibwedagama thank you for your recommend. I try using `<strong>`, this is my commit : https://github.com/jabardigitalservice/super-app-web/pull/7/commits/fcb16a3de8fb2538455910fe4b96eaf324dbd652